packages/studiocms_core/src/schemas/config/index.ts

```diff
@@ -74,6 +74,4 @@ export const StudioCMSOptionsSchema = z
 	.optional()
 	.default({});
 
-export type StudioCMSOptions = typeof StudioCMSOptionsSchema._input;
```

Comment:
Why did you remove this? it is used by other components of the CMS

Reply:
Knew I was gonna mess up during the rebase... lemme fix that really quick